src/cmdstan/stansummary.cpp

```diff
@@ -127,12 +128,10 @@ int main(int argc, const char* argv[]) {
   }
   
   // Prepare header
-  Eigen::Matrix<std::string, Eigen::Dynamic, 1> headers(n);
-  headers << 
-    "Mean", "MCSE", "StdDev",
-    "5%", "50%", "95%", 
-    "N_Eff", "N_Eff/s", "R_hat";
-  
+  std::vector<std::string> headers = {"Mean", "MCSE", "StdDev",
```

Comment:
Lists of strings like this shouldn't be duplicated.  But the bigger question is whether these change depending on the quantiles requested.  I wouldn't have thought 5, 50, and 95 could be locked in.

Reply:
Most of `print.cpp` is duplicate of `stansummary.cpp`, and is deprecated and already planned to be removed (#153). As such I don't think it's worth spending time to avoid duplication of those strings across the two files. But let me know what you think.

As for the hardcoding of percentiles, there's no way to request other percentiles through the current `stansummary` options (but should be doable via cmdstanr).

Reply:
OK.  I can just approve if it's already scheduled for demolition.